packages/DataPrivacy/src/Services/ProcessingActivityManager.php

```
@@ -291,4 +291,13 @@ private function generateActivityId(): string
     {
         return 'PA-' . date('Ymd') . '-' . strtoupper(substr(md5((string)microtime(true)), 0, 6));
     }
+
+    public function getActivitiesByDataSubject(\Nexus\DataPrivacy\ValueObjects\DataSubjectId $dataSubjectId): array
```

PHPMD (2.15.0)

[warning] Avoid unused parameters such as '$dataSubjectId'. (undefined)

(UnusedFormalParameter)